### PR TITLE
Update builder_image.md

### LINKED ISSUE #1083: guidelines url 404
this file (https://github.com/openshift/source-to-image/blob/master/docs/builder_image.md) uses this url - https://docs.okd.io/latest/creating_images/guidelines.html - that returns 404

### PR DESCRIPTION
**Pull request by coreydaley**
Fixes #1083

### CODE REVIEW

**Comment by coreydaley** [comment]
/assign @otaviof @akram 
Can one of you please lgtm this?

File: docs/builder_image.md [hunk 1]
```diff
@@ -177,4 +177,4 @@ should be the following:
 
 # Additional steps
 ## OpenShift support
-If you are intending to use this image with [OpenShift](https://github.com/openshift/origin), review and follow the OpenShift [image creation guidelines](https://docs.okd.io/latest/creating_images/guidelines.html).
+If you are intending to use this image with [OpenShift](https://github.com/openshift/origin), review and follow the OpenShift [image creation guidelines]([https://docs.okd.io/latest/creating_images/guidelines.html](https://docs.okd.io/latest/openshift_images/create-images.html)).
```
Comment (bparees) [inline docs/builder_image.md:180]: i don't think this is the edit you intended to make

Comment (coreydaley) [reply docs/builder_image.md]: Thanks for catching that, I used the `edit` feature on github, so maybe that messed it up and I pasted it.

**Pull request merged**